modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16447,6 +16447,7 @@ this-product-does-not-have-any-released-versions=This product does not have any
 this-product-is-discontinued.-you-can-see-the-replacement-product-by-clicking-on-the-button-below=This product is discontinued. You can see the replacement product by clicking on the button below.
 this-product-was-updated=This product was updated.
 this-product-will-be-automatically-updated-to-a-newer-version=This product will be automatically updated to a new version.
+this-publication-contains-changes-to-a-content-page-publishing-these-changes-will-fully-overwrite-the-page's-content-and-layout-in-production=This publication contains changes to a content page. Publishing these changes will fully overwrite the page's content and layout in production.
```

Comment:
Can you make sure there is a "." in the key? There should be a ton of examples in the file. Thanks!